crypto/o_str.c

```diff
@@ -280,12 +280,12 @@ char *ossl_buf2hexstr_sep(const unsigned char *buf, long buflen, char sep)
 
 /*
  * Given a buffer of length 'len' return a OPENSSL_malloc'ed string with its
- * hex representation @@@ (Contents of buffer are always kept in ASCII, also
- * on EBCDIC machines)
+ * hex representation with DEFAULT_SEPARATOR @@@ (Contents of buffer are always kept in ASCII, also
+ * on EBCDIC machines) 
```

Comment:
Please don't add end of line white space.

Reply:
I would just remove this comment change. The function is documented sufficiently in the corresponding .pod file.
If you want to change it maybe use ````using the separator DEFAULT_SEPARATOR``` and then
fix up the lines to not be larger than 80 chars.

Reply:
The end of line white space is removed.
The comment fall back, change 'len' to 'buflen' (<=80 chars).